src/test/java/com/wikia/webdriver/testcases/specialpagestests/EditingPreferencesTests.java

```diff
@@ -82,14 +82,12 @@ public void EditPreferences_003_selectSource() {
 
   @Test(groups = {"EditPreferences_004"})
   @Execute(asUser = User.USER_5)
-  @RelatedIssue(issueID = "MAIN-9722", comment = "test failing randomly")
   public void changeEmailAddress() {
     EditPreferencesPage editPrefPage = new EditPreferencesPage(driver).openEmailSection();
 
     String newEmailAddress = MailFunctions.getEmail(editPrefPage.getEmailAdress());
 
-    MailFunctions.deleteAllEmails(Configuration.getCredentials().emailQaart2,
-        Configuration.getCredentials().emailPasswordQaart2);
+    MailFunctions.deleteAllEmails(USERNAME, PASSWORD);
 
     Assertion.assertNotEquals(newEmailAddress, editPrefPage.getEmailAdress(),
```

Comment:
So is it 'mail' or 'email'? I'd prefer 'email' but let's be consistent (EmailFunctions)?